src/rust/ensogl/lib/text/src/component/area.rs

```diff
@@ -550,19 +884,51 @@ impl application::command::Provider for Area {
 
 impl application::View for Area {
     fn new(app:&Application) -> Self {
-        Area::new(&app.display.scene())
+        Area::new(app)
     }
 }
 
+// FIXME[WD]: Some of the shortcuts are commented out and some are assigned to strange key bindings
+//            because the shortcut manager is broken. To be fixed after fixing the shortcut manager.
 impl application::shortcut::DefaultShortcutProvider for Area {
-    fn default_shortcuts() -> Vec<application::shortcut::Shortcut> {
-        use enso_frp::io::keyboard::Key;
+    fn default_shortcuts() -> Vec<shortcut::Shortcut> {
         use enso_frp::io::mouse;
-//        vec! [ Self::self_shortcut(shortcut::Action::press (&[],&[mouse::PrimaryButton]), "set_cursor_at_mouse_cursor")
+//        vec! [ Self::self_shortcut(shortcut::Action::press (&[],&[mouse::PrimaryButton]), "set_cursor_at_mouse_position")
 //        ]
```

Comment:
commented code.

Reply:
There is description of why its commented - you have even commented this comment :D The comment was: 
```
// FIXME[WD]: Some of the shortcuts are commented out and some are assigned to strange key bindings
//            because the shortcut manager is broken. To be fixed after fixing the shortcut manager.
```